recipes/perl-vcftools-vcf/0.840/run_test.pl

```diff
@@ -0,0 +1 @@
+use Vcf;
```

Comment:
Consider conda/conda-build#1967. So the issue probably is fixed in conda-build 3.0.0.
How to proceed now with the pull request? Use the following?

```yaml
test:
  commands:
      - perl -e "use Vcf;"
```